analyses/MN_cd_2010/02_setup_MN_cd_2010.R

```diff
@@ -0,0 +1,22 @@
+###############################################################################
+# Set up redistricting simulation for `MN_cd_2010`
+# © ALARM Project, September 2022
+###############################################################################
+cli_process_start("Creating {.cls redist_map} object for {.pkg MN_cd_2010}")
+
+map <- redist_map(mn_shp, pop_tol = 0.005,
+    existing_plan = cd_2010, adj = mn_shp$adj)
+
+map <- map %>%
+    mutate(pseudo_county = pick_county_muni(map, counties = county, munis = muni,
+        pop_muni = 0.6*get_target(map)))
+
+# Add an analysis name attribute
+attr(map, "analysis_name") <- "MN_2010"
+
+# Fix labeling
+map$state <- "MN"
```

Comment:
can you move this to the prep file? no need to re-run any other code, just I think it belongs there